addons/point_of_sale/static/src/js/Screens/ClientListScreen/ClientListScreen.js

```diff
@@ -160,7 +161,7 @@ odoo.define('point_of_sale.ClientListScreen', function(require) {
                 this.state.detailIsShown = false;
                 this.render();
             } catch (error) {
-                if (error.message.code < 0) {
+                if (isRpcError(error) && error.message.code < 0) {
```

Comment:
Perhaps before doing this check, check first if the error is string like so:
```js
if (typeof error == 'string') {
    return this.showPopup('ErrorPopup', {
        title: this.env._t('Error'),
        body: error,
    });
}
```

Reply:
@fmdl